eslint.config.mjs

```diff
@@ -0,0 +1,36 @@
+import hmppsConfig from '@ministryofjustice/eslint-config-hmpps'
+
+export default [
+  // shared HMPPS standards
+  ...hmppsConfig,
+  
+  // app-specific overrides
+  {
+    ignores: ['**/node_modules', 'dist/', 'test_results'],
+  },
+  {
+    languageOptions: {
+      globals: {
+        ...hmppsConfig.find(obj => Object.keys(obj).includes('languageOptions')).languageOptions.globals,
+        $: false,
```

Comment:
This is a bit naff, but coping with jquery in asset files - previously we ignored asset files from eslint.

Is there a better way to do it?